docs/getting-started/create-server-cli.html

```diff
@@ -103,16 +103,37 @@ <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
     <div id="doc-container" class="docs-item-4 docs-window">
       <main>
         <h1 class="heading-1">Create a server with the Kitura CLI</h1>
-        <h2 class="heading-2" id="prereq">Prerequisites</h2>
-        <ul class="plain-list">
-          <li><a onclick="setActiveSidebarElementForParent('installation-mac')" href="../getting-started/installation-mac.html#install-kitura-cli">Kitura CLI:</a> Follow our instructions for installing the Kitura CLI.</li>
-        </ul>
+        <p>The Kitura command-line interface (CLI) is the fastest way to generate a blank Kitura project.</p>
```

Comment:
I don't like this wording when below we then say:
`kitura init` will create and build a cloud-ready Kitura application in your current directory. ???
I have reservations about the word "fastest" and I don't agree with the word "blank".